src/coreclr/jit/emitarm64.cpp

```diff
@@ -11731,31 +11872,87 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
                 vt              = tmpDsc->tdTempType();
             }
             if (vt == TYP_REF || vt == TYP_BYREF)
+            {
                 emitGCvarDeadUpd(adr + ofs, dst DEBUG_ARG(varNum));
+            }
         }
         if (emitInsWritesToLclVarStackLocPair(id))
         {
-            unsigned ofs2 = ofs + TARGET_POINTER_SIZE;
+            int      varNum2 = varNum;
+            int      adr2    = adr;
+            unsigned ofs2    = ofs;
+            unsigned ofs2Dist;
+
+            if (id->idIsLclVarPair())
+            {
+                bool FPbased2;
+                if (id->idSmallCns())
+                {
+                    varNum2 = ((instrDescLclVarPair*)id)->iiaLclVar2.lvaVarNum();
+                    ofs2    = ((instrDescLclVarPair*)id)->iiaLclVar2.lvaOffset();
+                }
+                else
+                {
+                    varNum2 = ((instrDescLclVarPairCns*)id)->iiaLclVar2.lvaVarNum();
+                    ofs2    = ((instrDescLclVarPairCns*)id)->iiaLclVar2.lvaOffset();
+                }
```

Comment:
My suggested `emitGetLclVarPairLclVar2()` could be used here, too.

Reply:
done.